packages/device_info/lib/device_info.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
What I would use _this_ plugin for is attach device/OS information to the error report.